android/BOINC/app/src/main/java/edu/berkeley/boinc/rpc/GlobalPreferences.java

```diff
@@ -92,15 +92,7 @@ public void writeToParcel(Parcel dest, int flags) {
         dest.writeValue(cpu_times);
         dest.writeValue(net_times);
 
-        dest.writeBooleanArray(new boolean[]{
-                run_on_batteries,
-                run_if_user_active,
-                run_gpu_if_user_active,
-                leave_apps_in_memory,
-                dont_verify_images,
-                override_file_present,
-                network_wifi_only
-        });
+        dest.writeBooleanArray(new boolean[]{run_on_batteries, run_if_user_active, run_gpu_if_user_active, leave_apps_in_memory, dont_verify_images, override_file_present, network_wifi_only});
```

Comment:
This is a type of change that makes things **clearly** worse. The human editor had clear and justified intentions (with the line breaks) which should be preserved.